### DISCUSSION TITLE
Timestamp in Discussions list is ambiguous

### DISCUSSION THREAD

**Discussion by prplecake** [community/community#76618] [Discussions]
### Select Topic Area

Product Feedback

### Body

Not sure if this should be in the profile section...

The timestamps on the list of ["your discussions"](https://github.com/discussions) is ambiguous since "m" is used for both minutes and months.

Turns out I don't use discussions much, and missed the single discussion that some X weeks ago separating the "X minutes ago" discussions from the "X months ago" discussions.

**Comment by Unmesh100** [comment]
You're absolutely right about the ambiguity with the "m" label for minutes and months in timestamps. It seems like a small detail, but it can cause confusion, especially when catching up on discussions. Clarity is key in user interfaces, and this is a prime example of where a minor tweak could make a significant difference.

It might be worth suggesting a change where "min" is used for minutes and "mo" for months. Simple, clear, and would save users from potential mix-ups.

If you haven't already, I'd recommend sharing this feedback directly with the platform's support or feedback channels. It’s these types of insights that drive meaningful improvements. Your attention to detail is valuable!
